src/Microsoft.PowerShell.Commands.Utility/commands/utility/ImplicitRemotingCommands.cs

```diff
@@ -2800,15 +2800,16 @@ private void GenerateHelperFunctions(TextWriter writer)
 
```

Comment:
@vexx32, your last commit had 1 failures in `PowerShell-CI-windows`
Remoting loopback tests.Can execute command in a disconnected session
```powershell
Objects cannot be added to a closed buffer. Make sure the buffer is open for Add and Insert operations to succeed.
at <ScriptBlock>, D:\a\1\s\test\tools\Modules\HelpersRemoting\HelpersRemoting.psm1: line 226
226:     Invoke-Command @parameters

```